src/arm64/arm_base64.cpp

```diff
@@ -325,6 +325,91 @@ void base64_decode_block(char *out, const char *src) {
   vst3q_u8((uint8_t *)out, outvec);
 }
 
+#if defined(_MSC_VER) && !defined(__clang__)
```

Comment:
Not for this PR, but it would be good if we factored out these bit-ops into a separate header/module.

Reply:
Agreed

Reply:
We already have a trailing_zeroes, so we can already simplify the code.

Reply:
Great!